wordpress-mobile/WordPress-iOS
# Build Warnings: remove different warnings

This PR gets rid of different build warnings. Every commit is a change towards fixing one warning, so I'd recommend reviewing commit by commit.

### To test

The only code changes that might affect the user are the images renaming.

1. Run Jetpack
2. Make sure that when you have unread notifications the icon displays correctly (dark and light mode)
3. Enable QS for an existing site (going to Settings)
4. Make sure the image has the Jetpack logo

Reproduce the same set of steps but for WordPress.

On Jetpack we're now at 13 warnings! 🥳 

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
